### PR TITLE
switcheroo: Add version 0.10.0-beta.1

### PR DESCRIPTION
**Pull request by snowman**

### CODE REVIEW

**Review by Ash258 — CHANGES_REQUESTED** [review]
The portable version does not persiste anything, which indicate two things:

1. It is not portable
1. You did not add it.

```powershell
remove-item bucket/switcheroo.json
git mv bucket/switcherro-portable.json bucket/switcheroo.json
```

**Comment by snowman** [comment]
Should it be below format

```json
{
    "version": "0.10.0-beta.1",
    "description": "The humble incremental-search task switcher for Windows",
    "homepage": "https://github.com/kvakulo/Switcheroo",
    "license": "GPL-3.0-only",
    "url": "https://github.com/kvakulo/Switcheroo/releases/download/v0.10.0-beta.1/switcheroo-setup.exe",
    "hash": "cca4227fd7fab5e908f18f78d2073b6f3fbbbd89599368c250c9e80a8d7ba9b9",
    "innosetup": true,
    "bin": "switcheroo.exe",
    "shortcuts": [
        [
            "switcheroo.exe",
            "switcheroo"
        ]
    ]
}
```

or just 
```powershell
remove-item bucket/switcheroo.json
git mv bucket/switcheroo-portable.json bucket/switcheroo.json
```

**Comment by Ash258** [comment]
What? The question does not make sense...

**Comment by snowman** [comment]
There are two versions: `switcheroo-setup.exe` and `switcheroo-portable.zip`

So if it's not portable, then I guess should download the `switcheroo-setup.exe` version instead of `git mv`?

I don't know how to contribute (have checked README.md), and I kind of don't know what inno-setup is. If you think either way is okay, then let it go. :p

**Comment by Ash258** [comment]
The portable as proposed in the two lines of powershell.

Keep it as it is.

File: bucket/switcheroo.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "hash": "cc46b8169946bed9d162e1514a17357960679365605e36449a144eb1660d8e61",
+    "bin": "switcheroo.exe",
+    "shortcuts": [
+        [
+            "switcheroo.exe",
+            "switcheroo"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/kvakulo/Switcheroo"
```
Comment (Ash258) [inline bucket/switcheroo.json:16]:
```suggestion
        "url": "https://github.com/kvakulo/Switcheroo/releases",
        "regex": "tag/v([\\w.-]+)"
```

File: bucket/switcheroo.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "url": "https://github.com/kvakulo/Switcheroo/releases/download/v0.10.0-beta.1/switcheroo-portable.zip",
+    "hash": "cc46b8169946bed9d162e1514a17357960679365605e36449a144eb1660d8e61",
+    "bin": "switcheroo.exe",
+    "shortcuts": [
+        [
+            "switcheroo.exe",
+            "switcheroo"
+        ]
+    ],
+    "checkver": {
```
Comment (Ash258) [inline bucket/switcheroo.json:15]:
```suggestion
    "persist": "switcheroo.settings",
    "checkver": {
```

File: bucket/switcheroo.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.10.0-beta.1",
+    "description": "The humble incremental-search task switcher for Windows",
+    "homepage": "https://github.com/kvakulo/Switcheroo",
+    "license": "GPL-3.0-only",
+    "url": "https://github.com/kvakulo/Switcheroo/releases/download/v0.10.0-beta.1/switcheroo-portable.zip",
+    "hash": "cc46b8169946bed9d162e1514a17357960679365605e36449a144eb1660d8e61",
+    "bin": "switcheroo.exe",
```
Comment (Ash258) [inline bucket/switcheroo.json:8]:
```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\switcheroo.settings\")) { New-Item \"$dir\\switcheroo.settings\" | Out-Null }",
```

File: bucket/switcheroo.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "shortcuts": [
+        [
+            "switcheroo.exe",
+            "switcheroo"
```
Comment (Ash258) [inline bucket/switcheroo.json:12]:
```suggestion
            "Switcheroo"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — COMMENTED** [review]
I would maybe remove the checkver and autoupdate since the repo is dead and PowerToys are released.

**Comment by rashil2000** [comment]
https://github.com/ScoopInstaller/Extras/commit/db2dba37be6cf95981e277a2087eb7edf0d1cc69

**Pull request closed without merging**